lib/event_tracker/tracker.rb

```diff
@@ -3,18 +3,33 @@ class Tracker
     def initialize(doer_id, context)
       @doer_id = doer_id
       @context = context
+      @trackers = []
+
+      register_tracker EventTracker::Trackers::Development.new(@doer_id) if development?
+      register_tracker EventTracker::Trackers::Mixpanel.new(@doer_id) if mixpanel?
+    end
+
+    def register_tracker(tracker)
+      @trackers << tracker
```

Comment:
should we check the tracker type before registering a new tracker?

Reply:
thoughts @oinak ?

Reply:
I think register_tracker could receive @doer_id and call itself a factory method like  `tracker_for` which only responsibility were instantiating the correct type.

Reply:
Might not even need to pass @doer_id around if it's already an instance variable (sorry for the fragmented responses, 23:30 here I'm on the mobile)